games-simulation/openrct2/openrct2-0.2.4.ebuild

```diff
@@ -0,0 +1,178 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-utils xdg-utils
+
+MY_PN="OpenRCT2"
+MY_PN_OBJ="objects"
+MY_PN_RPL="replays"
+MY_PN_TS="title-sequences"
+MY_PV_OBJ="1.0.12"
+MY_PV_RPL="0.0.4"
+MY_PV_TS="0.1.2c"
+
+DESCRIPTION="An open source re-implementation of Chris Sawyer's RollerCoaster Tycoon 2"
+HOMEPAGE="https://openrct2.org/"
+SRC_URI="
+	https://github.com/${MY_PN}/${MY_PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz
+	https://github.com/${MY_PN}/${MY_PN_OBJ}/releases/download/v${MY_PV_OBJ}/${MY_PN_OBJ}.zip -> ${PN}-${MY_PN_OBJ}-${MY_PV_OBJ}.zip
+	https://github.com/${MY_PN}/${MY_PN_TS}/releases/download/v${MY_PV_TS}/${MY_PN_TS}.zip -> ${PN}-${MY_PN_TS}-${MY_PV_TS}.zip
+	test? ( https://github.com/${MY_PN}/${MY_PN_RPL}/releases/download/v${MY_PV_RPL}/${MY_PN_RPL}.zip -> ${PN}-${MY_PN_RPL}-${MY_PV_RPL}.zip )
+"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="dedicated libressl +lightfx +opengl test +truetype twitch"
+
+COMMON_DEPEND="
+	dev-libs/icu:=
+	dev-libs/jansson:=
+	dev-libs/libzip:0=
+	media-libs/libpng:0=
+	sys-libs/zlib:0=
```

Comment:
Well, there is one slot, but also an subslot. Since you say, it was added wrongly in first place, where do I can read that, that QA says not to use that? Would help :-) And if it's wrong, why `sys-libs/zlib` is not corrected back?

Reply:
I've seen it explained somewhere but can't find it anymore. Or maybe zlogene told me.

Anyhow, here's the PR 
https://github.com/gentoo/gentoo/pull/5635
and there is much discussion in the forums about this. And as explained, removing that subslot will cause another similar shitshow as adding it. You can check for yourself that zlib hasn't changed soname.

Reply:
Allright. Thanks. It's sometimes really hard, if such information are only somewhere in a PR, but notwhere else :/ Since I already changed deps to `sys-libs/zlib`, this should be now fine.